src/arrayMethodIndexOf.js

```diff
@@ -6,7 +6,17 @@
  */
 function applyCustomIndexOf() {
   [].__proto__.indexOf2 = function(searchElement, fromIndex) {
```

Comment:
```suggestion
  [].__proto__.indexOf2 = function(searchElement, fromIndex = 0) {
```
Краще задавати default value.